jenkins/spark-premerge-build.sh

```diff
@@ -59,9 +59,10 @@ mvn_verify() {
               $MVN_CMD -B $MVN_URM_MIRROR -Dbuildver=$version test -rf tests $MVN_BUILD_ARGS -Dpytest.TEST_TAGS='' \
               -DwildcardSuites=org.apache.spark.sql.rapids.filecache.FileCacheIntegrationSuite
         # build only for other versions
-        elif [[ "${SPARK_SHIM_VERSIONS_NOSNAPSHOTS_TAIL[@]}" =~ "$version" ]]; then
-            $MVN_INSTALL_CMD -DskipTests -Dbuildver=$version
+        # elif [[ "${SPARK_SHIM_VERSIONS_NOSNAPSHOTS_TAIL[@]}" =~ "$version" ]]; then
+        #     $MVN_INSTALL_CMD -DskipTests -Dbuildver=$version
         fi
+        $MVN_INSTALL_CMD -DskipTests -Dbuildver=$SPARK_BASE_SHIM_VERSION
```

Comment:
Should this line be moved out of the for loop?

Reply:
The same question about `$MVN_INSTALL_CMD -DskipTests -Dbuildver=$SPARK_BASE_SHIM_VERSION`, otherwise BASE_SHIM will be build(without UT) multi times?

Can you share how long will verify, ci_2, scala_2.13 stages take?

Reply:
I've moved the step out of loop, and triggered again. I'll share the time-consuming when it's done.